examples/validate_relative_binding.py

```diff
@@ -181,28 +201,24 @@
     pressure = 1.0
     dt = 2.5e-3
 
+    cached_equil_path = "equil_{}_blocker_{}.pkl".format(
+        os.path.basename(cmd_args.protein_pdb).split(".")[0].replace(" ", "_"), cmd_args.blocker_name.replace(" ", "_")
```

Comment:
Seems a safe assumption that no directory names in path/to/proteinpdb contain a " ", not sure if we might want to do any other string -> filename sanitization here